ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift

```diff
@@ -8,22 +8,25 @@ import UIKit
 
 class DemoListViewController: UITableViewController {
 
-    static func addDemoListTo(window: UIWindow, pushing viewController: UIViewController?) {
-        if let colorProvider = window as? ColorProviding, let primaryColor = colorProvider.primaryColor(for: window) {
-            Colors.setProvider(provider: colorProvider, for: window)
-            FluentUIFramework.initializeAppearance(with: primaryColor, whenContainedInInstancesOf: [type(of: window)])
-        } else {
-            FluentUIFramework.initializeAppearance(with: Colors.primary(for: window))
-        }
+    private static var provider: ColorProviding = DemoColorDefaultTheme() as ColorProviding
```

Comment:
no need for being static right?

Reply:
provider variable is used in addDemoListTo and updateColorProviderFor, which are static methods. You cannot access non-static stuff in a static method.

Reply:
thanks for the explanation. now that you have change how window connects to demo controller, can we remove all these unnecessary static functions as much as possible?